natashaslprice/day2.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+	<meta charset="utf-8">
+
+	<meta name="viewport" content="width=devide-width, initial-scale=1">
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+	
+	<title>Day 2 homework</title>
+</head>
+
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-6">
+				<h1>Questions about homework:</h1>
+				<ul>
+					<li>I didn't understand Regular Expressions and why they are used. Please can you explain them?</li>
```

Comment:
@natashaslprice regular expressions are really powerful (often too powerful).  one example usage might be if we wanted to get all the user names from a bunch of emails
```js
function getUserName(email) { 
  return email.match(/[^@]+/)[0];  // match everything before the @ sig
}
emails = ['tash.price@gmail.com', 'travis@gmail.com', 'juliana@yahoo.com'];
emails.map(getUserName);   
```
this returns an array like: ["tash.price", "travis", "juliana"]